projects/nuvoton/numaker_iot_m487_wifi/uvision/aws_demos/aws_demos.uvproj

```diff
@@ -1,2346 +1,3673 @@
-						</File>
-						<File>
-							<FileName>oid.c</FileName>
-							<FileType>1</FileType>
-							<FilePath>../../../../../libraries/3rdparty/mbedtls/library/oid.c</FilePath>
-						</File>
-						<File>
-							<FileName>padlock.c</FileName>
-							<FileType>1</FileType>
-							<FilePath>../../../../../libraries/3rdparty/mbedtls/library/padlock.c</FilePath>
-						</File>
-						<File>
-							<FileName>pem.c</FileName>
-							<FileType>1</FileType>
-							<FilePath>../../../../../libraries/3rdparty/mbedtls/library/pem.c</FilePath>
-						</File>
-						<File>
-							<FileName>pk.c</FileName>
-							<FileType>1</FileType>
-							<FilePath>../../../../../libraries/3rdparty/mbedtls/library/pk.c</FilePath>
-						</File>
-						<File>
-							<FileName>pk_wrap.c</FileName>
-							<FileType>1</FileType>
-							<FilePath>../../../../../libraries/3rdparty/mbedtls/library/pk_wrap.c</FilePath>
-						</File>
-						<File>
-							<FileName>pkcs12.c</FileName>
-							<FileType>1</FileType>
-							<FilePath>../../../../../libraries/3rdparty/mbedtls/library/pkcs12.c</FilePath>
-						</File>
-						<File>
-							<FileName>pkcs5.c</FileName>
-							<FileType>1</FileType>
-							<FilePath>../../../../../libraries/3rdparty/mbedtls/library/pkcs5.c</FilePath>
-						</File>
-						<File>
-							<FileName>pkparse.c</FileName>
-							<FileType>1</FileType>
-							<FilePath>../../../../../libraries/3rdparty/mbedtls/library/pkparse.c</FilePath>
-						</File>
-						<File>
-							<FileName>pkwrite.c</FileName>
-							<FileType>1</FileType>
-							<FilePath>../../../../../libraries/3rdparty/mbedtls/library/pkwrite.c</FilePath>
-						</File>
-						<File>
-							<FileName>platform.c</FileName>
-							<FileType>1</FileType>
-							<FilePath>../../../../../libraries/3rdparty/mbedtls/library/platform.c</FilePath>
-						</File>
-						<File>
-							<FileName>platform_util.c</FileName>
-							<FileType>1</FileType>
-							<FilePath>../../../../../libraries/3rdparty/mbedtls/library/platform_util.c</FilePath>
-						</File>
-						<File>
-							<FileName>ripemd160.c</FileName>
-							<FileType>1</FileType>
-							<FilePath>../../../../../libraries/3rdparty/mbedtls/library/ripemd160.c</FilePath>
-						</File>
-						<File>
-							<FileName>rsa.c</FileName>
-							<FileType>1</FileType>
-							<FilePath>../../../../../libraries/3rdparty/mbedtls/library/rsa.c</FilePath>
-						</File>
-						<File>
-							<FileName>rsa_internal.c</FileName>
-							<FileType>1</FileType>
-							<FilePath>../../../../../libraries/3rdparty/mbedtls/library/rsa_internal.c</FilePath>
-						</File>
-						<File>
-							<FileName>sha1.c</FileName>
-							<FileType>1</FileType>
-							<FilePath>../../../../../libraries/3rdparty/mbedtls/library/sha1.c</FilePath>
-						</File>
-						<File>
-							<FileName>sha256.c</FileName>
-							<FileType>1</FileType>
-							<FilePath>../../../../../libraries/3rdparty/mbedtls/library/sha256.c</Fi
```

Comment:
Can we rename "M487_ETH_DEMO" to "M487_USE_ETHERNET" so that the test project can use this variable as well?

Reply:
@sarenameas  It's fine to me but this PR purpose is to add ETH lib and one simple demo sample for guidance. It seems unnecessary to link with test project, like as other boards with several  on-board connectivity. For ex: STM32 test project by WiFi but it still support ETH lib as: `\libraries\freertos_plus\standard\freertos_plus_tcp\source\portable\NetworkInterface\STM32Fxx` . pic32mzef also could support both of ETH & WiFi lib at: `\portable\NetworkInterface\pic32mzef`